platform/mellanox/hw-management/0003-Make-hw-mgmt-SimX-compatiable.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/usr/usr/bin/hw-management-ready.sh b/usr/usr/bin/hw-management-ready.sh
+index 3c9f7b6..22dc090 100755
+--- a/usr/usr/bin/hw-management-ready.sh
++++ b/usr/usr/bin/hw-management-ready.sh
+@@ -49,9 +49,12 @@ if [ -d /var/run/hw-management ]; then
+ 	rm -fr /var/run/hw-management
+ fi
+ 
+-while [ ! -d /sys/devices/platform/mlxplat/mlxreg-hotplug/hwmon ]
+-do
+-	sleep 1
+-done
++if [ "$(cat /sys/devices/virtual/dmi/id/sys_vendor)" != "QEMU" ]; then
++	while [ ! -d /sys/devices/platform/mlxplat/mlxreg-hotplug/hwmon ]
++	do
++		sleep 1
```

Comment:
@shlomibitton why do we need to sleep? Why not simply return? Does it mean that hw-management service will always be in starting condition?

Reply:
The loop is polling to check if the hw-mgmt sysfs created before proceeding with the flow.
AFAIK, we want to wait until it is valid before exiting this script, this is the previous implementation.
The fix is only to skip it if this is a SimX platform.

Reply:
@shlomibitton `"$(cat /sys/devices/virtual/dmi/id/sys_vendor)" != "QEMU"` this doesn't work anymore. Please use `lspci` instead.